src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp

```diff
@@ -101,7 +119,24 @@ PrimListUnpackReplacer::PrimListUnpackReplacer() {
 
             return true;
         }
+        if (auto where = cast_fw_node(input_node, "aten::nonzero_numpy")) {
```

Comment:
Better to rename `where` variable here, it looks like the name is just a copy artifact.

Reply:
renamed